components/button/demo/size.md

```diff
@@ -1,24 +1,28 @@
 ---
 order: 2
-title: 按钮尺寸
+title:
+  zh-CN: 按钮尺寸
+  en-US: Size
 ---
 
+## zh-CN
+
 按钮有大、中、小三种尺寸。
 
 通过设置 `size` 为 `large` `small` 分别把按钮设为大、小尺寸。若不设置 `size`，则尺寸为中。
 
+## en-US
+
+Thers are large button, default button and small button in antd.
+
+To get large/small button, just set `size` as `large`/`small`. If nothing is provided to `size`, we will get default button.
```

Comment:
Ant-design supports a default button size as well as a large and small size.

If a large or small button is desired, set the `size` attribute to either `large` or `small` respectively. Omit the `size` attribute for a button with the default size.